### DISCUSSION TITLE
Search Bar Broken

### DISCUSSION THREAD

**Discussion by RizonFTW** [InfinityBotList/.github#8] [Bug Reports]
On the latest update (4.9.0) I have noticed the search bar is broken. 

Console states that error code 500

Video and screenshots below

https://github.com/InfinityBotList/.github/assets/30637943/01ee5783-97a8-489c-80de-3daf5e5d4d89

![image](https://github.com/InfinityBotList/.github/assets/30637943/8ac22db3-e539-4730-8ad7-81ea1e42b646)

**Comment by cheesycod** [comment]
Oops, will fix
